hw-6 (weatherstack)/script.js

```diff
@@ -0,0 +1,29 @@
+let button = document.getElementById("button");
+
+button.onclick = function () {
```

Comment:
We are using `addEventListener` instead of such a way.
Here is a good description why: https://stackoverflow.com/questions/6348494/addeventlistener-vs-onclick